examples/sugarscape_g1mt/sugarscape_g1mt/agents.py

```diff
@@ -303,13 +305,15 @@ def trade(self, other):
             sold = self.maybe_sell_spice(other, price, welfare_self, welfare_other)
             if not sold:
                 return
+            self.prices.append(price)
         else:
             # self is a spice buyer
             sold = other.maybe_sell_spice(self, price, welfare_other, welfare_self)
             if not sold:
                 return
+            self.prices.append(price)
```

Comment:
You should just de-dent this line, and remove the previous `self.prices.append`, because this append happens in all cases regardless of the if-else situation.

Reply:
Will do